api.go

```diff
@@ -168,8 +168,8 @@ type UserLoginResponse struct {
 type PostCommandRequest struct {
 	Issuer           string                   `json:"issuer"`
 	Actions          []map[string]interface{} `json:"actions"`
-	Schema           string                   `json:"schema"`
-	SchemaVersion    int                      `json:"schemaVersion"`
+	Schema           string                   `json:"schema,omitempty"`
+	SchemaVersion    int                      `json:"schemaVersion,omitempty"`
```

Comment:
Changing these two props to `omitempty` means that
make`PostCommand()` allows those props omittable.

If it is OK, there are no problems.
Otherwise it would be better to declare `PostTraitCommandRequest` type.